feathr_project/feathr/client.py

```diff
@@ -434,6 +442,13 @@ def get_offline_features(self,
             _FeatureRegistry.save_to_feature_config_from_context(self.anchor_list, self.derived_feature_list, self.local_workspace_dir)
         else:
             raise RuntimeError("Please call FeathrClient.build_features() first in order to get offline features")
+        
+        # Pretty print anchor list and derived_feature_list
+        if verbose:
+            if self.anchor_list:
+                FeaturePrinter.pretty_print(self.anchor_list)
+            if self.derived_feature_list:
+                FeaturePrinter.pretty_print(self.derived_feature_list)
```

Comment:
we should only print feature_query. Feature anchor is a super set of feature_query. 

For example, i have an anchor and it contains f1, f2, f3.

But I only want f1 and f2. Then my feature_query = f1, f2.

Giving out whole anchors is confusing.

Reply:
@hangfei  
Ok no problem. I can build a PrettyPrinter for `feature_query`. Just one clarification. (Really sorry for the questions)
1. Is `feature_query` supposed to be printed only in `get_offline_features` or should it be printed in `materialize_features` and `build_features` as well?
1. If `feature_query` needs to be printed in `materialize_features` and `build_features`, there is no `feature_query` argument in `materialize_features` and `build_features`. How can `feature_query` be referred in `materialize_features` and `build_features`?
https://github.com/linkedin/feathr/blob/7dd36a5496c301006136a0dde5caf07306d26464/feathr_project/feathr/client.py#L213
https://github.com/linkedin/feathr/blob/7dd36a5496c301006136a0dde5caf07306d26464/feathr_project/feathr/client.py#L530